infrastructure/general/setup-bootnode.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Set Up a Bootnode
+description: Learn how to configure and run a bootnode for Polkadot, including P2P, WS, and secure WSS connections with network key management and proxies.
+---
+
+# Set Up a Bootnode
+
+## Introduction
+
+Bootnodes are essential for helping blockchain nodes discover peers and join the network. When a node starts, it needs to find other nodes, and bootnodes provide an initial point of contact. Once connected, a node can expand its peer connections and play its role in the network, like participating as a validator. This guide will walk you through setting up a Polkadot bootnode, configuring P2P, WebSocket (WS), and secure WSS connections, and managing network keys. You’ll also learn how to test your bootnode to ensure it is running correctly and accessible to other nodes.
+
+## Prerequisites
+
+Before you start, you need to have the following prerequisites:
+
+<!-- TODO: What goes here? -->
```

Comment:
@0xLucca or @nhussein11: please add the appropriate items here. We have a nice example in `tutorials/polkadot-sdk/build-a-blockchain/add-trusted-nodes.md` on #29 so you can see what I mean. Thank you!

Reply:
I added a few, but please see if it needs to change.